sunpy/net/dataretriever/sources/gong_synoptic.py

```diff
@@ -0,0 +1,103 @@
+from sunpy.net.dataretriever import GenericClient
+from sunpy.util.scraper import Scraper
+
+__all__ = ['GongSynopticClient']
+
+
+class GongSynopticClient(GenericClient):
+    """
+    Provides access to the Magnetogram products of NSO-GONG synoptic Maps.
+
+    Searches data hosted by the `National Solar Observatory. <gong2.nso.edu/oQR/zqs/>`__
+
+    Examples
+    --------
+    >>> from sunpy.net import Fido, attrs as a
+    >>> results = Fido.search(a.Time("2019/12/31 22:00", "2020/1/1 02:00"),
+    ...                       a.Instrument('GONG'))  #doctest: +REMOTE_DATA
+    >>> results  #doctest: +REMOTE_DATA
+    <sunpy.net.fido_factory.UnifiedResponse object at ...>
+    Results from 1 Provider:
+    <BLANKLINE>
+    3 Results from the GongSynopticClient:
+         Start Time           End Time      Source Instrument Wavelength
+    ------------------- ------------------- ------ ---------- ----------
+    2019-12-31 22:00:00 2020-01-01 02:00:00    NSO       GONG        nan
+    2019-12-31 22:00:00 2020-01-01 02:00:00    NSO       GONG        nan
+    2019-12-31 22:00:00 2020-01-01 02:00:00    NSO       GONG        nan
```

Comment:
This result looks odd, do you know why this is picking up three files, but all with the same timestamp?

Reply:
Fixed now.